server/plugin/plugin.go

```diff
@@ -175,6 +175,7 @@ func (p *Plugin) OnConfigurationChange() (err error) {
 		e.bot = e.bot.WithConfig(stored.Config)
 		e.Dependencies.Remote = remote.Makers[gcal.Kind](e.Config, e.bot)
 
+		// REVIEW: need to make this provider agnostic terminology
 		mscalendarBot := mscalendar.NewMSCalendarBot(e.bot, e.Env, pluginURL)
```

Comment:
mscalendar terminology leaked into `plugin` package. not much else than what's visible here though